masterlist.yaml

```
@@ -1672,6 +1672,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'DisabletheCaravan.esl'
```

There's an ESP version on the page that'd benefit from the same group assignment:

```suggestion
  - name: 'DisabletheCaravan\.es[lp]'
```